src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/import_data.cljs

```diff
@@ -29,6 +29,7 @@
                                (set! (.-onload reader)
                                      #(swap! state assoc :file file :file-contents (.-result reader)))
                                (.readAsText reader (.slice file 0 preview-limit)))))}]
+      [comps/PHIWarning]
```

Comment:
Do we need this one here?  This causes it to be shown twice in the import data dialog.